code/datums/status_effects/debuffs.dm

```diff
@@ -824,3 +824,101 @@
 
 /datum/status_effect/transient/drugged/on_remove()
 	owner.update_druggy_effects()
+
+#define FAKE_COLD 1
+#define FAKE_FOOD_POISONING 2
+#define FAKE_RETRO_VIRUS 3
+#define FAKE_TURBERCULOSIS 4
+
+/datum/status_effect/fake_virus
+	id = "fake_virus"
+	duration = 3 MINUTES
+	status_type = STATUS_EFFECT_REPLACE
+	tick_interval = 2
+	alert_type = null
+	/// So you dont get the most intense messages immediately
+	var/msg_stage = 0
+	/// Which disease we are going to fake?
+	var/current_fake_disease
+	/// Fake virus messages by with three stages
+	var/list/fake_msg
+	/// Fake virus emotes with three stages
+	var/list/fake_emote
+
+/datum/status_effect/fake_virus/on_creation()
+	current_fake_disease = pick(FAKE_COLD, FAKE_FOOD_POISONING, FAKE_RETRO_VIRUS, FAKE_TURBERCULOSIS)
+	switch(current_fake_disease)
+		if(FAKE_COLD)
+			fake_msg = list(
+						list("<span class='danger'>Your throat feels sore.</span>", "<span class='danger'>Mucous runs down the back of your throat.</span>"),
+						list("<span class='danger'>Your muscles ache.</span>", "<span class='danger'>Your stomach hurts.</span>"),
+						list("<span class='danger'>Your muscles ache.</span>", "<span class='danger'>Your stomach hurts.</span>")
+			)
+			fake_emote = list(
+						list("sneeze", "cough"),
+						list("sneeze", "cough"),
+						list("sneeze", "cough")
+			)
+		if(FAKE_FOOD_POISONING)
+			fake_msg = list(
+						list("<span class='danger'>Your stomach feels weird.</span>", "<span class='danger'>You feel queasy.</span>"),
+						list("<span class='danger'>Your stomach aches.</span>", "<span class='danger'>You feel nauseous.</span>"),
+						list("<span class='danger'>Your stomach hurts.</span>", "<span class='danger'>You feel sick.</span>")
+			)
+			fake_emote = list(
+						list(null),
```

Comment:
```suggestion
						list(),
```
You can use `safepick` instead so you don't have to provide a list with a null dummy value